lesson6/README.md

```diff
@@ -0,0 +1,87 @@
+# Lesson 6 - How to Create an App
+
+##  Designing an app
+
+#### 1. Observe
+Open any app on your phone. It could be very simple or very complex; there is something to learn from both. While using it, ask questions. Some examples of questions could be:
+  1. What does the logo/name communicate to the users?
+  2. Are all the features easily discoverable?
+  3. In order to perform an action, how many times did you tap? Too many popups asking you to choose YES or NO?
+  4. How does the color scheme fit the overall feel?
```

Comment:
how about "how easy are certain buttons to press? do you press those buttons often?"